plugins/user_quota/girder_user_quota/quota.py

```diff
@@ -258,7 +258,10 @@ def _getBaseResource(self, model, resource):
                 return None, None
         if model == 'file':
             model = 'item'
-            resource = Item().load(id=resource['itemId'], force=True)
+            if 'attachedToId' in resource:
+                resource = Item().load(id=resource['attachedToId'], force=True)
```

Comment:
I don't think we track the size usage of the attached files (and this isn't necessarily an item).

Reply:
Alright, I can try to exit gracefully when an attached item is logged here. I'm somewhat unfamiliar with the design behind the user quota plugin, but when I tried to exit similarly in the past it got unhappy. I'll give it a shot and see if there are errors

Reply:
Turned out this wasn't the issue I was thinking of. The new code works and avoids quota logic for attached files entirely